src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java

```diff
@@ -320,10 +324,11 @@ else if (checkName.equals(currentConfig.getName())) {
 
     private static String removeDeviceFromPathOnWindows(String path) {
         final String os = System.getProperty("os.name", "Unix");
+        String fixedPath = path;
```

Comment:
We could make this `final` and put `path` into `else` statement.
@romani Would you prefer it this way?